### PR TITLE
bugfix: Resolve accidential toggling on My Recipes button

### LINKED ISSUE #104: Fix MyRecipes button toggle issue

### PR DESCRIPTION
**Pull request by jsahim**
Files changed: scripts.js
Change summary: I updated the showSavedRecipes function to include a conditional that only allows it's code to execute if savedView is assigned to false. 
Additional Questions/Concerns: potential merge conflict

#104

### CODE REVIEW

**Review by LordSchwifty — APPROVED** [review]
This should work. Nice fix!

File: src/scripts.js [hunk 1]
```diff
@@ -173,16 +173,18 @@ function getRandomUser() {
...
-  const savedRecipes = currentUser.recipesToCook.map(userRecipe => {
-    return recipeRepo.recipes.find(recipe => recipe.id === userRecipe)
-  });
-  currentView = savedRecipes;
-  searchBar.placeholder = 'Search My Recipes...';
-  filterHeader.innerText = 'Filter My Recipes';
-  toggleHidden(myRecipesTitle);
-  toggleHidden(titleLogo);
-  createRecipeCards(currentView);
+  if(!savedView){
```
Comment (qrispi) [inline src/scripts.js:176]: Live, Laugh, Love this Fix!

**Review by qrispi — APPROVED** [review]
Thanks Jeff!

File: src/scripts.js (hunk 1 above)
Comment (jfogiato) [reply src/scripts.js:176]: MIND BLOWING SOLUTION. DOCTORS HATE IT. CHECK OUT THIS ONE EASY CONDITIONAL TO MAKE YOUR CODE LAST FOR HOURS

**Pull request merged**